CMakeLists.txt

```diff
@@ -52,20 +52,20 @@ include(CMakeDependentOption)
 # Setup shared library / -fPIC stuff
 get_property(SHARED_LIBS_SUPPORTED GLOBAL PROPERTY TARGET_SUPPORTS_SHARED_LIBS)
 cmake_dependent_option(ADIOS_BUILD_SHARED_LIBS
-  "Whether or not to build shared libraries" ON
+  "Whether or not to build shared libraries" OFF
```

Comment:
We use the .a library as default as in adios1, I think it has to do with some systems having trouble with so libraries.